src/Core/Analysis/QuickFix.cs

```diff
@@ -49,19 +49,22 @@ public class Edit : IEdit
             string text)
         {
             StartLine = startLine;
-            StartColumn = startColumn;
+            StartLineOffset = startColumn;
```

Comment:
Renamed to match the properties in ITextRange (that were based on the existing IAnalysisIssueLocation)